src/testrender/testrender.cpp

```diff
@@ -279,7 +279,7 @@ main (int argc, const char *argv[])
         rend->clear();
         delete shadingsys;
         delete rend;
-    } catch (const optix::Exception& e) {
+    } catch (const OSL::optix::Exception& e) {
```

Comment:
Without the `OSL` namespace, both gcc and clang fail for me in Linux with this error: `ISO C++ forbids declaration of ‘optix’ with no type [-Werror=permissive]`. Since this change was necessary for me to compile and test, I decided to tack it onto this review.